src/telemetry/telemetry.c

```diff
@@ -16,19 +25,59 @@
 
 #define MODULE_COUNT 2
 
+static downlink_node *downlink_queue = NULL;
+
 /* This list controls the order of initialisation */
 static const module_init_t init_sequence[MODULE_COUNT] = {
-    {"downlink", &init_downlink},
-    {"downlink_queue", &init_downlink_queue}
+        {"downlink",       &init_downlink},
+        {"downlink_queue", &init_downlink_queue}
 };
 
-int init_telemetry( void ){
+// TODO: For test purposes only!
+void *test_telemetry_generator() {
+    for (int i = 0; i < 100; ++i) {
+        send_telemetry_local(&downlink_queue, i, 0);
+    }
+    return NULL;
+}
+
+/**
+ * Infinite loop for sending telemetry via UDP.
+ *
+ * @return
+ *
+ * @TODO Move it to `init_telemetry`?
+ * @TODO Redo it all!!!
+ */
+void *telemetry_sender() {
```

Comment:
Designwise, it was intented that what this thread does would be the responsibility of the `downlink` module.

Reply:
As you wrote, I'll move contents of `downlink` into `e-link` and I can move this into `downlink` then.